src/websocket/types/room/objects/extension.rs

```diff
@@ -12,9 +13,10 @@ with_structure_fields_and_update_struct! {
         #[serde(default, rename = "off")]
         pub disabled: bool,
         /// The current amount of energy held in this structure.
-        pub energy: i32,
+        pub store: Store,
         /// The maximum amount of energy that can be held in this structure.
-        pub energy_capacity: i32,
+        #[serde(rename = "storeCapacityResource")]
+        pub capacity_resource: Store,
```

Comment:
What would you think of keeping the original `store_capacity_resource` name? I realize naming decisions are a bit fuzzy here, especially since this library renames other arbitrary things. My thinking here would be that since the things we're storing are part of a `store` now, rather than on the structure itself, it makes more sense to qualify the `capacity` variable with `store_` as well.

Reply:
This is a good point. I just committed the change.